hw/ip_templates/otp_ctrl/dv/env/otp_ctrl_scoreboard.sv.tpl

```diff
@@ -906,6 +995,76 @@ else:
                   end
                 end
               end
+              DaiZeroize: begin
+                bit[TL_AW-1:0] otp_addr = get_scb_otp_addr();
+
+                bit part_zero_addr_seen = 0;
+                otp_part_addr_cov_e offset_addr_cov;
+
+                `uvm_info(`gfn, $sformatf("Zeroizing Loc: Part Index :%s, PartZeroFieldAddr :%x",
+                                      part_idx.name, PART_OTP_ZEROIZED_ADDRS[part_idx]), UVM_LOW);
+                `uvm_info(`gfn, $sformatf("Zeroizing Loc: otp_addr :%x, dai_addr :%x",
+                                      otp_addr, dai_addr), UVM_LOW);
+                `uvm_info(`gfn, $sformatf("Zeroizing Loc: is_granule_64 :%x",
+                                      is_granule_64(dai_addr)), UVM_LOW);
+
+                // Need to check if Partition is zeroizable first. If not raise interrupt and set
+                // the status and error registers
+                if (!part_is_zeroizable(int'(part_idx))) begin
+                  predict_err(OtpDaiErrIdx, OtpAccessError);
+
+                  if (cfg.en_cov) begin
+                    cov.zr_dai_cmd_cg.sample(.part_idx(part_idx),
+                                             .zeroizable(0),
+                                             .offset_addr(OtpPartStartAddr));
+                  end
+                end else begin
+                  bit [TL_DW*2-1:0] descrambled_val;
+                  bit [TL_DW-1:0]   rdata_ret_val = '{default:1};
```

Comment:
'{default:1} is equivalent to '1 for a regular vector, and the latter is preferable.